### ISSUE TITLE
Support for multi-pass encoding

### ISSUE THREAD

**Issue by escaped** [escaped/django-video-encoding#11]
Many encoders provide multi-pass encoding for higher image quality and compression ration (eg. [VP9](https://developers.google.com/media/vp9/settings/vod/#ffmpeg_command_lines)). This is currently not supported and would be a nice addition for some use cases.

**Comment by ajeema** [comment]
if you want to use multipass, although there are much better methods that are faster, a tweak would be to reuse the first pass log file.

**Comment by escaped** [comment]
yeah, that's basically why it isn't supported. Currently we only do one pass without being able to reuse the logs for a second pass :) Pull Requests are welcome!
